package.json

```diff
@@ -2,6 +2,7 @@
   "name": "protocol",
   "version": "0.1.0",
   "description": "UMA Protocol Smart Contracts",
+  "private": true,
```

Comment:
Why did you change this to private?

Reply:
Yarn workspaces require it for the root `package.json`: https://classic.yarnpkg.com/en/docs/workspaces/#toc-how-to-use-it.